fastlane/Fastfile

```diff
@@ -125,7 +126,13 @@ private_lane :appstore_api_key do
 end
 
 lane :pod_lint do
-  pod_lib_lint(podspec: 'StreamChatSwiftUI.podspec', allow_warnings: true)
+  dependencies = JSON.parse(File.read(project_package_resolved))
+  is_branch = dependencies["pins"].find { |dependency| dependency['identity'] == 'stream-chat-swift' && dependency['state']['branch'] }
+  if is_branch.nil?
+    pod_lib_lint(podspec: 'StreamChatSwiftUI.podspec', allow_warnings: true)
+  else
+    UI.important('Pod linting skipped: stream-chat-swift targets a development branch')
+  end
```

Comment:
I'm curious, does the current solution mean that we have to build the package first to generate `Package.resolved` to check if `pod lint` is required? As far as I know, `Package.resolved` is in `.gitignore` and is not pushed to `origin/`.

If so, maybe we could just parse `StreamChatSwiftUI.xcodeproj/project.pbxproj` via [xcodeproj](https://github.com/CocoaPods/Xcodeproj) and read the dependency rule straight away? 🤔

Reply:
Did not figure it out so I added resolve packages step instead.